interceptor/tracing_interceptor.go

```diff
@@ -374,6 +388,7 @@ func (t *tracingWorkflowInboundInterceptor) HandleSignal(ctx workflow.Context, i
 			runIDTagKey:      info.WorkflowExecution.RunID,
 		},
 		FromHeader: true,
+		Time:       time.Now(),
```

Comment:
Will we need a directive comment here to tell the determinism analysis plugin that this can be ignored?

Reply:
Hrmm, I doubt it because I think the code traversal for analysis doesn't walk past the "temporal boundary" which interceptors sit behind. May be worth checking though.